packages/frontend/src/components/CFSignin.vue

```diff
@@ -6,101 +8,105 @@
         name="mdi-check-circle-outline"
         size="16"
         fill="var(--vscode-notebookStatusSuccessIcon-foreground, #388a34)"
-      ></v-mdi>
+      />
       You are signed in to Cloud Foundry.
     </div>
     <br /><br />
 
     <div :style="{ display: loggedInVisibility }">
       <span class="subtitle-color-field">Cloud Foundry Endpoint</span>
       <br />
-      <div class="mt-8">{{ endpoint }}</div>
+      <div class="mt-8">
+        {{ endpoint }}
+      </div>
       <br /><br />
-      <vscode-button @click="SignoutClicked">Sign Out</vscode-button>
+      <vscode-button @click="SignoutClicked"> Sign Out </vscode-button>
     </div>
 
     <!-- authentication area -->
-    <div :style="{ display: notLoggedInVisibility }" id="authenticationDiv">
+    <div id="authenticationDiv" :style="{ display: notLoggedInVisibility }">
       <br />
       <span class="subtitle-color-field">Enter Cloud Foundry Endpoint </span><span class="text-danger">*</span>
       <br />
       <vscode-text-field
         id="cfEndpointInput"
+        ref="cfendpointInput"
+        :value="endpoint"
         class="pt-8"
         :class="{ 'invalid-input-field': !isCFEndpointValid }"
         size="50"
         type="url"
-        ref="cfendpointInput"
-        :value="this.target.defaultEndpoint"
         @input="setEndpoint"
-        v-on:keyup="btnStatus"
-      ></vscode-text-field>
+        @keyup="btnStatus"
+      />
       <div v-if="!isCFEndpointValid" class="error-container">
         <div class="invalid-endpoint-error">You must provide a valid URL</div>
       </div>
       <br /><br />
       <span class="subtitle-color-field">Select authentication method </span>
-
-      <v-mdi
-        name="mdi-help-circle-outline"
-        align-self=""
-        size="16"
-        fill="var(--vscode-textLink-foreground, #006ab1)"
+      <span
         v-tooltip="{
-          content:
-            ' Single sign-on (SSO) is a token-based authentication method <br />in which an SSO token is passed in an HTTP header or cookie.',
-          placement: 'right',
-          class: 'tooltip-custom',
-          size: '5',
+          text:
+            'Single sign-on (SSO) is a token-based authentication method in which an SSO token is passed in an HTTP header or cookie.',
+          theme: {
+            placement: 'right',
+            width: '300px',
```

Comment:
consider to define and use an unique css class instead of list of properties (like it was before - on left panel). IMHO it is a easier way to maintenance it

Reply:
The list of properties is an API you should use to adjust the tooltip  - https://github.com/maciejziemichod/v-tooltip/tree/main#api-reference